src/main/java/RacingCar.java

```diff
@@ -0,0 +1,30 @@
+public class RacingCar {
+    /**
+     * 자동차는 이름을 가지고 있다.
+     * 자동차는 움직일 수 있다.
+     * 0에서 9 사이에서 random 값을 구한 후 random 값이 4 이상일 경우 전진하고, 3 이하의 값이면 멈춘다.
+     */
+    // 자바 네이밍 규칙에 따라 상수는 대문자로 작성
+    public static final int FIXNUMBER = 3; // 기준값 선언
```

Comment:
대문자로 작성해주실 때 언더바 `_` 로 단어별 구분을 해주면 보다 읽기 편한 변수명 짓기를 할 수 있습니다 :)

```suggestion
    public static final int FIXED_NUMBER = 3; // 기준값 선언
```